### DISCUSSION TITLE
Firewall On - Docker Server Closed Unexpectedly

### DISCUSSION THREAD

**Discussion by beck0644** [itzg/docker-minecraft-bedrock-server#309] [Q&A] [answered]
My bedrock server won't start when the firewall on my Synology NAS is on.  Upon starting, the log hangs at "Looking up latest version..." then I receive the message "Docker server closed unexpectedly".  
![Screenshot 2023-03-25 203808](https://user-images.githubusercontent.com/108092974/227750557-8590367a-03e5-4a92-9f12-3be58264e547.jpg)

I reviewed the restify.err file and found, "2023/03/25 20:29:48 Failed to load content: Failed to retrieve response: Get "https://www.minecraft.net/en-us/download/server/bedrock": dial tcp: lookup www.minecraft.net on 127.0.0.11:53: read udp 127.0.0.1:56515->127.0.0.11:53: i/o timeout"

When the firewall is off the bedrock server starts and runs just fine.  I've also found that I can start the server with the firewall off then turn the firewall on and it runs just fine.

I'd like to figure this out so I don't have to turn the firewall off every time the server resets.

**Comment by itzg** [comment, marked as answer]
Timeout to port 53 means there is specifically DNS communication being blocked. Look into adjusting your firewall instead of turning it fully off. If that's not possible then try configuring the container with an external DNS such as CloudFlare's 1.1.1.1.

**Reply by beck0644** [reply]
Thanks!
